src/main/java/frc/trigon/robot/subsystems/climber/ClimberConstants.java

```diff
@@ -0,0 +1,157 @@
+                    RETRACTED_CLIMBER_LENGTH_METERS,
+                    MAXIMUM_HEIGHT_METERS,
+                    SIMULATE_GRAVITY
+            );
+
+    static final SysIdRoutine.Config SYSID_CONFIG = new SysIdRoutine.Config(
+            Units.Volts.of(1.5).per(Units.Second.of(1)),
+            Units.Volts.of(8),
+            null,
+            null
+    );
+
+    static final Translation3d
+            RIGHT_CLIMBER_FIRST_JOINT_ORIGIN_POINT = new Translation3d(0.1, 0, 0.1), //TODO: get numbers from mechanics
+            LEFT_CLIMBER_FIRST_JOINT_ORIGIN_POINT = new Translation3d(0.1, 0, 0.1); //TODO: get numbers from mechanics
+    static final Translation3d
+            RIGHT_CLIMBER_SECOND_JOINT_ORIGIN_POINT = new Translation3d(0, 0, 0.1), //TODO: get numbers from mechanics
+            LEFT_CLIMBER_SECOND_JOINT_ORIGIN_POINT = new Translation3d(0, 0, 0.1); //TODO: get numbers from mechanics
+    static final ElevatorMechanism2d
+            RIGHT_MECHANISM = new ElevatorMechanism2d(
+            "RightClimberMechanism", MAXIMUM_HEIGHT_METERS, RETRACTED_CLIMBER_LENGTH_METERS, new Color8Bit(Color.kRed)
+    ),
+            LEFT_MECHANISM = new ElevatorMechanism2d(
+                    "LeftClimberMechanism", MAXIMUM_HEIGHT_METERS, RETRACTED_CLIMBER_LENGTH_METERS, new Color8Bit(Color.kRed)
+            );
+
+    static final Rotation2d STRING_PITCH = Rotation2d.fromDegrees(75); //TODO: get number from mechanics
+    static final double STRING_LENGTH_METERS = 0.2; //TODO: get number from mechanics
+    static final Pose3d STRING_POSE = new Pose3d(0, 0, 0, new Rotation3d(0, 0, 0));
+
+    static {
+        configureMotor(RIGHT_MOTOR, RIGHT_MOTOR_INVERTED_VALUE, RIGHT_MOTOR_SIMULATION);
+        configureMotor(LEFT_MOTOR, LEFT_MOTOR_INVERTED_VALUE, LEFT_MOTOR_SIMULATION);
+    }
+
+    private static void configureMotor(TalonFXMotor motor, InvertedValue invertedValue, ElevatorSimulation simulation) {
+
+        final TalonFXConfiguration config = new TalonFXConfiguration();
+
+        config.MotorOutput.Inverted = invertedValue;
+        config.MotorOutput.NeutralMode = NEUTRAL_MODE_VALUE;
+        config.Audio.BeepOnBoot = false;
+        config.Audio.BeepOnConfig = false;
+
+        config.Slot0.kP = P;
+        config.Slot0.kI = I;
+        config.Slot0.kD = D;
+        config.Slot0.kS = KS;
+        config.Slot0.kV = KV;
+        config.Slot0.kG = KG;
+        config.Slot0.kA = KA;
+
+        config.Feedback.SensorToMechanismRatio = GEAR_RATIO;
+
+        motor.applyConfiguration(config);
+        motor.setPhysicsSimulation(simulation);
+
+        motor.registerSignal(TalonFXSignal.POSITION, 100);
+        motor.registerSignal(TalonFXSignal.CLOSED_LOOP_REFERENCE, 100);
+        motor.registerSignal(TalonFXSignal.STATOR_CURRENT, 100);
+    }
+
+    public enum ClimberState {
+        RETRACTED(0),
+        CLIMBING(0.7), //TODO: get number from mechanics
+        RESTING(0.2); //TODO: get number from mechanics
```

Comment:
there will be 3 states. 
RESTING(0)
PREPARE_FOR_CLIMBING(x) // TODO: ~get number from mechanics~ calibrate
CLIMBING(x) // TODO: ~get number from mechanics~ calibrate

Add a boolean to whether or not this state is affected by gravity or not. Look at 2024's climber for reference and why that's needed.